src/commands/config-setup.js

```diff
@@ -0,0 +1,58 @@
+const handleErrors = require("../utils/handleErrors");
+const jsYaml = require('js-yaml')
+const fs = require('fs')
+const inquirer = require("inquirer")
+
+module.exports.command = "config-setup";
+module.exports.describe =
+    "Enters customizable configurations into config file";
+module.exports.builder = (yargs) => yargs;
+const CONFIG_URL = 'templates/infra-authentication-server/config/defaultroles.yml'
+
+const questions = [
+    {
+      type: "confirm",
+      name: "gauth",
+      message:
+        "Would you like to use Google Authentication for signing in?"
+    },
+    {
+      type: "confirm",
+      name: "gmail",
+      message: "Would you like to use Gmail for password recovery?"
+    },
+    {
+      type: "confirm",
+      name: "sq",
+      message: "Would you like to use a security question for password recovery?"
+    }
+];
+
+let responses = ["", "", ""];
+
+askQuestions = async () => {
+    for(let i = 0; i < questions.length; i++){
+        const response = await inquirer.prompt(questions[i])
+        responses[i] = response[Object.keys(response)[0]]
+    }
+}
+
+module.exports.handler = handleErrors(async (argv) => {
+    try {
+        console.log("Note that Google Authentication and Gmail both require setting up Google Oauth.")
+        await askQuestions();
+    } catch (e) {
+    return console.error(e);
+    }
+    if (!responses[1] && !responses[2])
```

Comment:
Instead of this, lets have them pick one of three options (gmail, security question, or both)